src/main/java/dansplugins/factionsystem/MedievalFactions.java

```diff
@@ -49,7 +51,7 @@ public class MedievalFactions extends PonderBukkitPlugin {
     private final PlayerTeleporter playerTeleporter = new PlayerTeleporter(logger);
     private final TerritoryOwnerNotifier territoryOwnerNotifier = new TerritoryOwnerNotifier(configService.getLocaleService(), configService, actionBarService);
     public boolean USE_NEW_LANGUAGE_FILE = configService.getBoolean("useNewLanguageFile");
-    private MedievalFactions medievalFactions;
+    private MedievalFactions medievalFactions = this;
     private final MessageService messageService = new MessageService(medievalFactions);
```

Comment:
We can just pass `this` into the services, we needn't use a reference within the class itself

Reply:
We are now passing 'this' into the necessary services.